static/css/src/_m-ip-select.scss

```diff
@@ -155,7 +155,9 @@ Styleguide SelectMenu.UIStates
         user-select: none;
 
         &:nth-child(even) {
-            background-color: #E6EEF1;
+            --alt-row-bg: #E6EEF1;
```

Comment:
This can be higher up in the `block` scope. In the same level as the `--opt-bg--hover` variable.